src/support/utf8.c

```diff
@@ -232,7 +232,14 @@ size_t u8_offset(const char *s, size_t charnum)
     return i;
 }
 
-/* byte offset => charnum */
+/**
+ * @brief      Calculates number of characters in a UTF-8 string
+ * 
+ * @param[in]  iStr UTF-8 encoded string
```

Comment:
In my own utf.h/utf.c, I was following the convention agreed upon at the Belgian startup I'm consulting for... we are using Julia, at least in parts (but increasingly, people want to use it everywhere ;-) )